kubernetes-deployment/template/kubelet.sh.template

```diff
@@ -43,4 +43,6 @@ docker run \
   --pod-manifest-path=/etc/kubernetes/manifests \
   --allow-privileged=true \
   --logtostderr=true \
+  --image-gc-high-threshold=98 \
```

Comment:
>image-gc-high-threshold=98 [](start = 4, length = 26)

[https://github.com/kubernetes/kubernetes/pull/51423](https://github.com/kubernetes/kubernetes/pull/51423) in the future we may use this.